static/app/views/projectsDashboard/index.tsx

```diff
@@ -36,101 +36,96 @@ type Props = {
   error: Error | null;
 } & RouteComponentProps<{orgId: string}, {}>;
 
-class Dashboard extends Component<Props> {
-  componentWillUnmount() {
-    ProjectsStatsStore.reset();
-  }
-
-  render() {
-    const {teams, params, organization, loadingTeams, error} = this.props;
-
-    if (loadingTeams) {
-      return <LoadingIndicator />;
-    }
+function Dashboard({teams, params, organization, loadingTeams, error}: Props) {
+  useEffect(() => {
+    return function cleanup() {
+      ProjectsStatsStore.reset();
+    };
+  }, []);
 
-    if (error) {
-      return <LoadingError message="An error occurred while fetching your projects" />;
-    }
+  if (loadingTeams) {
+    return <LoadingIndicator />;
+  }
 
-    const filteredTeams = teams.filter(team => team.projects.length);
-    filteredTeams.sort((team1, team2) => team1.slug.localeCompare(team2.slug));
+  if (error) {
+    return <LoadingError message="An error occurred while fetching your projects" />;
```

Comment:
Was this not translated before?

Reply:
Nope! But I'll add a translation wrapper now.